crash_similarity.py

```diff
@@ -70,6 +70,18 @@ def get_stack_trace_from_crashid(crash_id):
 def get_stack_traces_for_signature(fnames, signature):
     traces = set()
 
+    #query stack traces online
+    url = 'https://crash-stats.mozilla.com/api/SuperSearch'
+    params = {
+        'signature': '=' + signature,
+        '_columns':  'proto_signature'
+    }
```

Comment:
The result will be in `res.json()['facets']` instead of `res.json()['hits']`, see for example: https://crash-stats.mozilla.com/api/SuperSearch/?signature=%3Djs::jit::ICStub::trace&_facets=proto_signature.

Also, you can add `_results_number=0` to make the query less expensive: https://crash-stats.mozilla.com/api/SuperSearch/?signature=%3Djs::jit::ICStub::trace&_facets=proto_signature&_results_number=0.